pepper-apis/src/main/java/org/broadinstitute/ddp/constants/ConfigFile.java

```diff
@@ -133,6 +135,8 @@
     public static final String DSM_JWT_SIGNER = "dsmJwtSigner";
     public static final String USE_DISPOSABLE_TEST_DB = "useDisposableTestDbs";
 
+    public static final String AUTH0_IP_WHITE_LIST = "auth0IpWhiteList";
+
```

Comment:
Our 2 new config parameters: `restrictRegisterRoute` and `auth0IpWhiteList` . Note that I purposely left `auth0IpWhiteList` outside of the Auth0 config parameters that appear defined lower in this ConfigFile class as those other ones appear to be solely used for testing purposes.